### ISSUE TITLE
Fails with ipywidgets from conda-forge

### ISSUE THREAD

**Issue by scottyhq** [jacobtomlinson/gha-anaconda-package-version#7]
This action has been workhorse for pangeo metapackages for over a year! @jacobtomlinson and @jhamman thanks so much for putting it together. 

Not sure how to diagnose the issue but trying to use this action for `ipywidgets` on `conda-forge` is failing:
```
     - name: Get latest ipywidgets version
       id: latest_version_ipywidgets
       uses: jacobtomlinson/gha-anaconda-package-version@0.1.2
       with:
         org: "conda-forge"
         package: "ipywidgets"
```
https://github.com/pangeo-bot/dispatcher/runs/2266803393?check_suite_focus=true 

`conda list -c conda-forge ipywidgets` doesn't show any unusual version tags (e.g._): 
```
ipywidgets                     7.4.2            py_0  conda-forge         
ipywidgets                     7.5.0            py_0  conda-forge         
ipywidgets                     7.5.1            py_0  conda-forge         
ipywidgets                     7.5.1    pyh9f0ad1d_1  conda-forge         
ipywidgets                     7.6.0    pyhd3deb0d_0  conda-forge         
ipywidgets                     7.6.2    pyhd3deb0d_0  conda-forge         
ipywidgets                     7.6.2    pyhd3deb0d_1  conda-forge         
ipywidgets                     7.6.3    pyhd3deb0d_0  conda-forge  
```

The logs are massive with 10,000+ lines I think coming from leftover debug statement from #5  : https://github.com/jacobtomlinson/gha-anaconda-package-version/blob/c5aa7cd32d5ec258be94df1d26fdb85fc66b7368/main.go#L62

**Comment by jacobtomlinson** [comment]
> This action has been workhorse for pangeo metapackages for over a year! @jacobtomlinson and @jhamman thanks so much for putting it together.

🎉. I created this to automate the Dask Docker image, glad it's been useful elsewhere!

> The logs are massive with 10,000+ lines

Ouch, yup thats a problem.

> Catch incompatible semver version strings on conda-forge #8

Thanks for raising this PR @scottyhq. I've merged and released `0.1.3`. Do you want to give it another go?

**Comment by scottyhq** [comment]
switched to 0.1.3 over in https://github.com/pangeo-bot/dispatcher and I think all is well!  thanks @jacobtomlinson !
